bloggies_frontend/src/Forms/LoginForm/index.tsx

```diff
@@ -29,7 +29,7 @@ function LoginForm({ loginUser }: IProp) {
       <div className="LoginForm-wrapper">
         <p className="LoginForm-title text-left">Welcome back!</p>
         <Form className="LoginForm-form" onSubmit={handleSubmit}>
-          <Form.Control className="form-input" name="username" value={formData.username} placeholder="Username" onChange={handleChange} required></Form.Control >
+          <Form.Control className="form-input" name="email" value={formData.email} placeholder="Username" onChange={handleChange} required></Form.Control >
```

Comment:
I think you missed the `username` in the `placeholder attribute`. xD

Reply:
Oh I did 👍 thanks I'll update that.

Reply:
Updated it.